application_form/services/offer.py

```diff
@@ -87,3 +94,16 @@ def update_reservations_based_on_offer_expiration(
         reservation.set_state(ApartmentReservationState.OFFERED, user=user)
 
     return new_expired_reservations.count(), not_anymore_expired_reservations.count()
+
+
+def update_other_reservation_state_in_queue(reservation):
```

Comment:
I guess `reservation` in the name could be `reservations` to indicate better that we are updating all of them. ⛏️

Reply:
Yeah, updated